src/models/TestStationDAO.ts

```diff
@@ -43,11 +44,25 @@ export class TestStationDAO {
   }
 
   /**
-   * Get All test stations in the DB
+   * Get All test stations in the DB, filtered by status
```

Comment:
see defects ... you may need to conditionally disable xray

Reply:
Requested change completed.